src/Page/Options.elm

```diff
@@ -271,21 +201,54 @@ viewResultItemDetails channel item =
 
                 Just value ->
                     wrapWith value
+
+        wrapped wrapWith value =
+            case value of
+                "" ->
+                    wrapWith <| "\"" ++ value ++ "\""
+
+                _ ->
+                    wrapWith value
+
+        showDetails =
+            if Just item.source.name == show then
+                [ div []
+                    [ div [] [ text "Default value" ]
+                    , div [] [ withEmpty (wrapped asPreCode) item.source.default ]
+                    , div [] [ text "Type" ]
+                    , div [] [ withEmpty asPre item.source.type_ ]
+                    , div [] [ text "Example" ]
+                    , div [] [ withEmpty (wrapped asPreCode) item.source.example ]
+                    , div [] [ text "Declared in" ]
+                    , div [] [ withEmpty asGithubLink item.source.source ]
+                    ]
+                ]
+
+            else
+                []
```

Comment:
Not sure if it's worth it but I would consider rather returning `Maybe (Html msg)` here and using `List.filterMap identity` to get rid off nothing. I think there is even function for this in `list-extra`.